### DISCUSSION TITLE
tutorials/article/keycloak-identity-server-sozlash

### DISCUSSION THREAD

**Discussion by giscus** [ismoilovdevml/devops-journey#41] [General]
# tutorials/article/keycloak-identity-server-sozlash

Keycloak bilan Identity Server sozlash

https://devops-journey.uz/tutorials/article/keycloak-identity-server-sozlash

**Comment by BSardorbek** [comment]
redis cache bilan integratsiya qilsa boladimi Keycloakni ?

**Reply by ismoilovdevml** [reply]
ha integratsiya qilsa bo'ladi

https://www.keycloak.org/server/caching
